src/hooks/use-api-error.hook.ts

```diff
@@ -6,20 +6,20 @@ import { useOnSignOutApp } from './app/use-on-sign-out-app.hook'
 type HandleApiError = (err: unknown) => any
 
 export const useApiErrorHook = (): HandleApiError => {
-  const handleSignOutApp = useOnSignOutApp()
+  const onAPiErrorHook = useOnSignOutApp()
 
-  const handleApiError: HandleApiError = useRecoilCallback(
+  const onApiErrorHook: HandleApiError = useRecoilCallback(
```

Comment:
Make sure that the type has the same name

HandleApiError => OnApiErrorHook

Reply:
Handled in [6015b91](https://github.com/ajktown/wordnote/pull/158/commits/6015b9172668f39d6487836b62c2d30bdb1bf421)